traffic_ops/app/lib/API/Cachegroup.pm

```diff
@@ -242,11 +253,19 @@ sub create {
 		return $self->alert( "A cachegroup with short_name " . $short_name . " already exists." );
 	}
 
+	my $coordinate_values = {
+		name      => "from_cachegroup_" . $params->{name},
```

Comment:
what is the string concatenation of `from_cachegroup_` about?

Reply:
I added that prefix so that it was obvious when viewing coordinates which ones belong to cachegroups, rather than just giving it the same name as the cachegroup.

Reply:
Yea I wouldn't force that as a convention if it is a freeform field because it's too easy for Users to enter what they want.  If it's truly a linkage that we need then it should be formalized as a relational database linkage

Reply:
Yeah it's mainly just for the migration of existing cachegroups into the new schema. We have to create Coordinates from the existing Cachegroups then set the Cachegroup.Coordinate FK to the newly created Coordinate ID. The Coordinate ID is looked up using "from_cachegroup_<cachegroup.name>", so that's the only place where the "linkage" from Coordinate to Cachegroup via the name is needed.